.github/workflows/docker-hub-parachain.yml

```diff
@@ -11,6 +11,15 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
 
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            zeitgeistpm/zeitgeist-node-parachain
+          tags: |
+            type=sha
```

Comment:
Will deploy a new image based on the commit sha hash

Reply:
Does it also push to latest?

Reply:
It will now. Thanks